### PR TITLE
Improve the phone number sanitization

### PR DESCRIPTION
**Pull request by valanz**

### CODE REVIEW

File: spec/Scoringline/SendinblueApi/Api/SmsSpec.php [hunk 1]
```diff
@@ -135,4 +135,10 @@ function it_should_throw_error_if_api_fails_while_send_campaign(TransformerInter
             ->duringSendCampaign(12, '+33685965788')
         ;
     }
+
+    function it_should_sanitize_phone_number()
+    {
+        $this->sanitizePhoneNumber('+1 234-324-6532')->shouldReturn('12343246532');
```
Comment (Nek-) [inline spec/Scoringline/SendinblueApi/Api/SmsSpec.php:141]: Should return "001", isn't it ?

File: spec/Scoringline/SendinblueApi/Api/SmsSpec.php [hunk 2]
```diff
@@ -135,4 +135,10 @@ function it_should_throw_error_if_api_fails_while_send_campaign(TransformerInter
...
+        $this->sanitizePhoneNumber('+33(0)62453-27-32')->shouldReturn('330624532732');
```
Comment (Nek-) [inline spec/Scoringline/SendinblueApi/Api/SmsSpec.php:142]: Should return "0033', right ?

File: spec/Scoringline/SendinblueApi/Api/SmsSpec.php (hunk 1 above)
Comment (Nek-) [inline spec/Scoringline/SendinblueApi/Api/SmsSpec.php:141]: https://www.wikiwand.com/fr/Liste_des_indicatifs_t%C3%A9l%C3%A9phoniques_internationaux_par_pays

Comment (valanz) [reply spec/Scoringline/SendinblueApi/Api/SmsSpec.php:141]: @Nek- we don't have to pass these "00" to sendinblue. They will add them themselves.

```
+1 310-962-3135

array(3) {
  'code' =>
  string(7) "success"
  'message' =>
  string(14) "Data retrieved"
  'data' =>
  array(7) {
    'status' =>
    string(2) "OK"
    'number_sent' =>
    int(1)
    'to' =>
    string(13) "0013109623135"
    'sms_count' =>
    int(1)
    'credits_used' =>
    int(1)
    'remaining_credit' =>
    double(40249.5)
    'reference' =>
    array(1) {
      [1] =>
      string(17) "dtdjqv7qblrmvaewa"
    }
  }
}
```

Comment (valanz) [reply spec/Scoringline/SendinblueApi/Api/SmsSpec.php:141]: @Nek- https://apidocs.sendinblue.com/mailin-sms/#1

**Pull request merged**